packages/forms/resources/views/components/checkbox-list.blade.php

```diff
@@ -88,20 +89,23 @@
                             'dark:border-gray-600' => (! $errors->has($getStatePath())) && config('forms.dark_mode'),
                             'border-danger-300 ring-danger-500' => $errors->has($getStatePath()),
                             'dark:border-danger-400 dark:ring-danger-400' => $errors->has($getStatePath()) && config('forms.dark_mode'),
+                        ])->merge([
+                            'x-on:change' => $isBulkToggleable() ? 'updateIsAllSelected' : null
                         ]) }}
-                    />
+                        />
 
-                    <span @class([
+                        <span @class([
```

Comment:
Here as well.

Reply:
I think I got it right now. Please review the indentions again.